changelog.md

```diff
@@ -1,4 +1,4 @@
-# 1.2 - xxxx-xx-xx
+# 1.1 - xxxx-xx-xx
```

Comment:
Don't change this.

Reply:
Okay, didn't know about that. So this means all `{.since: (1, 1).}` pragmas in the codebase are wrong too and should be updated?

Reply:
> So this means all {.since: (1, 1).} pragmas in the codebase are wrong too and should be updated?
  
No, v1.1 is the current devel. And if you change the since pragma to 1.2, then those procs won't be usable in devel.

Reply:
> So this means all {.since: (1, 1).} pragmas in the codebase are wrong too and should be updated?

No, as @nc-x said, they are correct and should be left as they are.

Reply:
Okay fine, I'll do as instructed but I'm not quite getting this new versioning scheme... Is there a summary somewhere about it so I can understand it?

Reply:
Odd *patch* versions were traditionally devel versions, e.g. 0.18.1, 0.19.3, 1.0.5, etc.

We have adopted similar versioning for post-1.0 *minor* versions, e.g. 1.1.1 is current devel version.
The reason behind such decision is exactly stuff like this where we can do `since (x, y)` and it can be already tested with devel (version `x.y`) and will be included in the next stable release (version `x.y+1`).

Reply:
Thanks for the explanation, I suspected this after thinking a bit about it and it makes sense. It's similar to the NodeJS versioning scheme.